distribution/docs/src/main/resources/_contents/_securing/authentication-components-contents.adoc

```diff
@@ -0,0 +1,114 @@
+
+==== Authentication Components
+
+It is assumed that the three authentication components identified below are installed on three separate servers.
```

Comment:
Hm. I only see two components. However, there is nothing wrong with running the LDAP and STS on the same server. We might just need to reword this a bit. We don't package up an LDAP that we intend people to use, because that introduces a lot of risk into DDF. The LDAP server needs to be configured correctly and probably conform to enterprise rules. For that reason, it makes a lot of sense to just run an LDAP server (such as an Active Directory instance) on a machine that is separate from the one running DDF. 

We also talk about running an STS only installation, which is certainly something that you can do if you have a cluster of DDFs that you want to have authenticate against a single STS, but I'm not sure that this is a standard scenario. I would think that a configuration like that would be better served under documentation for any high availability support that we might come up with in the future.